portal/templates/portal/base.html

```diff
@@ -244,12 +244,13 @@
     });
 
     function animateScroll() {
+        var bannerHeight = $('.sticky-subnav').height();
```

Comment:
is it possible to do this only once per page load?

Reply:
I'll try...